src/test/ui/issues/issue-76191.stderr

```diff
@@ -14,8 +14,30 @@ LL |         RANGE => {}
    |
    = note: expected type `i32`
             found struct `RangeInclusive<i32>`
+help: you may want to move the range into the match block
+   |
+LL | const RANGE: RangeInclusive<i32> = 0..=255;
+   |                                    ^^^^^^^
```

Comment:
As mentioned above, this suggestion should be pointing at `RANGE`, not the span we take the snippet from.